File: app/src/main/java/com/example/dcct/bean/RecordHistory.java
package com.example.dcct.bean;

public class RecordHistory {
    private int gaugingKind;
    private String[] thingName;
    private String gaugingTime;

    public RecordHistory() { }

    public RecordHistory(int gaugingKind, String[] thingName, String gaugingTime) {
        this.gaugingKind = gaugingKind;
        this.thingName = thingName;
        this.gaugingTime = gaugingTime;
    }

    public void setGaugingKind(int gaugingKind) {
        this.gaugingKind = gaugingKind;
    }

    public void setThingName(String[] thingName) {
        this.thingName = thingName;
    }

    public void setGaugingTime(String gaugingTime) {
        this.gaugingTime = gaugingTime;
    }

    public int getGaugingKind() {
        return gaugingKind;
    }

    public String[] getThingName() {
        return thingName;
    }

    public String getGaugingTime() {
        return gaugingTime;
    }
}

File: app/src/main/java/com/example/dcct/bean/RecordHistory.kt
package com.example.dcct.bean

class RecordHistory {
    var gaugingKind = 0
    lateinit var thingName: Array<String>
    var gaugingTime: String? = null

    constructor() {}
    constructor(gaugingKind: Int, thingName: Array<String>, gaugingTime: String?) {
        this.gaugingKind = gaugingKind
        this.thingName = thingName
        this.gaugingTime = gaugingTime
    }

}